### ISSUE TITLE
Add automated analysis support for pytorch models

### ISSUE THREAD

**Issue by chebroluharika** [flow2ml/Flow2ML#109]
Create a class for automated evaluation of pytorch models that will accept the model's history as an input and make the roc curves, confusion matrix and pr curves. The class can have an additional method to create a doc file with those plots.

**Comment by chebroluharika** [comment]
@ArkaprabhaChakraborty Please goahead and work on this.

**Comment by ArkaprabhaChakraborty** [comment]
Okay on it.

**Comment by yvkrishna** [comment]
@ArkaprabhaChakraborty, when can we expect output for this ?

**Comment by ArkaprabhaChakraborty** [comment]
I'm having my exams till the 27th. So maybe after that

**Comment by ArkaprabhaChakraborty** [comment]
Alright so after checking and trying out different scripting codes. I found out the best way to do this for pytorch has been already solved in a previously merged PR. Pytorch allows more tweaking and easier implementations of models with fewer calls to prebuilt functions like tensorflow. So there is almost no shortcut to this process for pytorch. One can use the y_pred and y_true label arrays as an input to Auto_Results() object.
